code/modules/mob/death.dm

```diff
@@ -12,8 +12,9 @@
 /mob/proc/spawn_gibs()
 	hgibs(loc)
 
-
-
+///Proc called when siloing a mob
+/mob/proc/siloed(obj/structure/resin/silo/siloed_to)
+	return
```

Comment:
Mob/living, the rest shouldnt get silod anyway

Reply:
That removes the need for a cast down the line, since the variable of attack by of the silo is mob, not mob/living

Reply:
Then cast it as living in the attackby

Reply:
done